tasks/stage-product/task.sh

```diff
@@ -24,7 +24,7 @@ function main() {
 
   local version
   pushd "${cwd}/pivnet-product"
-    version="$(ls -1 *.pivotal | sed "s/"${PRODUCT_NAME}"-\(.*\).pivotal/\1/")"
+    version="$(unzip -p *.pivotal 'metadata/*.yml' | cat | grep 'product_version:' | cut -d ':' -f 2 | tr -d ' ')"
```

Comment:
Kris, please take another look. I have removed the redundant 'cat' but also added a `tr` command in the end of that line to remove single quotes from product version fields, as Synchrony Financial ran into that issue with the New Relic tile, which contains single quotes around the version number and makes the `om` command fails (as version not found).